src/modules/transport-mode/icon/index.tsx

```diff
@@ -56,27 +57,57 @@ export function TransportMonoIcon({ mode }: TransportIconProps) {
 export type TransportIconWithLabelProps = {
   mode: TransportModeGroup;
   label?: string | null;
+  duration?: number | null;
```

Comment:
Does this really need to have both undefined AND null?

Reply:
No, probably not. Same with label I suppose, though in that case the `publicCode` (which is the main use for the label) is nullable and can be .

Reply:
Removed `null` from both. A bit more verbose when passing the label though.